Project/Engine/src/sysFrame/Object/Light/DirectionalLight/DirectionalLight.cpp

```
@@ -0,0 +1,26 @@
+#include "DirectionalLight.h"
+
+#include "imgui.h"
+
+void RawDirectionalLight::DefaultSetting() {
+    type_ = LightType::Directional;
+    light_.color = {1, 1, 1, 1};
+    light_.direction = {0, -1, 0};
+    light_.intensity = 1.f;
+}
+
+void RawDirectionalLight::ImGuiSetting() {
```

Cppcheck (2.10-2)

[style] The function 'ImGuiSetting' is never used.

(unusedFunction)